make-hash.js

```diff
@@ -1,4 +1,7 @@
 'use strict'
+if (typeof Buffer === 'undefined') {
+  global.Buffer = require('buffer').Buffer
```

Comment:
I have already patched bitcoinjs-lib [here](https://github.com/39otrebla/react-native-bitcoinjs-lib), but RN packager was still complaining about "Cannot find variable Buffer". I am not sure why all modules recognize Buffer but yours, but it is what happened. And I solved only by applying the "shim" for Buffer directly there, as above.